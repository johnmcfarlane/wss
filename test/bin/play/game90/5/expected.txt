 49  1,9  - LIDE
 43  1,9  - GID
 42 11,1  | GILSE
 40 10,2  | GLADII
 40  1,9  - LID
 40  2,1  | REGILD
 40  2,1  | REGILL
 40  2,1  | RIDGEL
 40  2,1  | RIDGIL
 39 11,1  | GEDS
 39 11,1  | GELS
 39 11,1  | GIDS
 39 11,1  | GILS
 39 11,1  | GLIS
 36 10,2  | EGALL
 36 11,1  | GIES
 36 10,2  | GLADE
 36 10,2  | GLALI
 36  2,1  | RILLED
 33 11,1  | DIGS
 33 11,1  | LEGS
 33  3,7  | YODEL
 33  3,7  | YODLE
 31  2,9  - IDE
 30 11,1  | DELS
 30 11,1  | LEDS
 30 11,1  | LIDS
 30 11,4  | SIGILL
 27 11,1  | DEIS
 27 11,1  | DIES
 27  9,3  - ELI
 27 11,1  | LEIS
 27 11,1  | LIES
 24  1,2  - DEIL
 24  9,1  | DIERI
 24 11,1  | EGIS
 24 11,1  | ELDS
 24 11,1  | ELLS
 24 10,3  | GALLIED
 24 11,2  | GISEL
 24  2,9  - ID
 24 11,1  | ILLS
 23  1,9  - GI
 22 10,3  | GALILEI
 22 10,3  | GALLED
 22  4,5  - GLiDE
 21 11,2  | DISLI
 21  1,1  | EDGE
 21  1,1  | EDILE
 21  1,1  | ELIDE
 21  9,1  | GEIR
 21 11,1  | IDES
 21 11,3  | ISLED
 21 11,2  | LISLE
 21  9,1  | LLER
 20 10,4  | ALGEDI
 20  1,9  - DI
 20 10,3  | GALEID
 20  9,3  | GRILLED
 20  1,9  - LI
 20  6,6  - NILLED
 19  3,9  - DE
 19  5,5  - LiDE
 18 10,4  | ADIGEI
 18 10,4  | ALGID
 18 10,4  | ALLIED
 18 10,3  - DE
 18 10,3  - DI
 18  9,1  | DIER
 18  1,1  | EILD
 18  1,1  | ELLE
 18 10,3  | GALLI
 18 11,2  | GDS
 18 11,2  | GISE
 18 10,1  | GLEAD
 18 10,1  | GLIAL
 18 10,1  | LEGAL
 18 10,3  - LI
 18  9,1  | LIER
 18  6,6  - NIDGE
 18  6,6  - NIELLI
 18  6,6  - NIGEL
 17  4,3  - DECL
 17  1,2  - DEI
 17  5,3  - LCD
 17  1,2  - LEI
 17  6,5  - iDE
 16 10,4  | AGIEL
 16 10,4  | AGILE
 16 10,3  | DALLE
 16  7,7  | EILD
 16 10,3  | GALEI
 16 10,3  | GALL
 16 10,2  | GLAD
 16  9,3  | GRILLE
 16 10,3  | LADLE
 16  2,1  | REDIG
 16  2,1  | RELIG
 16  2,1  | RIDGE
 16  9,4  | RIDGEL
 16  9,4  | RIDGIL
 16  2,1  | RIGEL
 16  2,1  | RIGID
 15 11,2  | DESI
 15  1,1  | EIDE
 15  1,1  | ELD
 15  7,7  | ELD
 15  1,1  | ELL
 15  7,7  | ELL
 15 11,2  | GES
 15 11,2  | GIS
 15  9,3  | GRILL
 15 11,3  | ISED
 15 11,3  | ISLE
 15  8,4  | JELL
 15  8,4  | JILL
 15 11,2  | LISE
 15 11,3  | LSD
 14 10,4  | ADIEL
 14 10,4  | AGED
 14 10,4  | AILED
 14 10,4  | ALLIE
 14 10,2  | DIAG
 14  9,2  | DIRGIE
 14  6,2  - EDILE
 14 10,2  | EGAD
 14 10,2  | EGAL
 14  6,2  - ELIDE
 14 10,3  | GADE
 14 10,3  | GADI
 14 10,3  | GAED
 14 10,3  | GAEL
 14 10,3  | GAIL
 14 10,3  | GALE
 14 10,3  | GALI
 14 10,2  | GEAL
 14  3,5  - GELiD
 14 10,1  | GILA
 14  9,2  | GIRDLE
 14 10,1  | GLIA
 14 10,1  | IDEAL
 14 10,2  | IGAD
 14 10,1  | ILEAL
 14 10,1  | ILIAD
 14 10,1  | ILIAL
 14  8,4  | JIG
 14  2,1  | REGD
 14  9,4  | REGILD
 14  9,4  | REGILL
 14  2,1  | RILED
 14  2,1  | RILLE
 14  9,4  | RILLED
 13  1,2  - DE
 13  9,3  | DRILL
 13  9,5  - ELI
 13  7,7  | ELI
 13  4,2  - GEED
 13  3,3  - GEIC
 13  4,2  - GIED
 13  9,2  | GIRLIE
 13  9,3  | GRIDE
 13  4,2  - LIEGE
 13  3,1  | SIGIL
 12  4,1  | AGILE
 12 10,4  | AILIE
 12 10,4  | ALG
 12 10,4  | ALLE
 12 10,3  | DALE
 12 10,3  | DALI
 12 10,2  | DEAL
 12 10,2  | DIAL
 12  9,2  | DIRGE
 12  9,2  | DIRIGE
 12 11,2  | DIS
 12  1,1  | EEL
 12  9,2  | EGRID
 12  1,1  | ELI
 12 10,1  | ELLA
 12 11,2  | ELS
 12  3,2  - IDEE
 12 11,2  | IDS
 12 11,3  | ISL
 12 10,3  | LADE
 12 10,3  | LAID
 12 10,2  | LEAD
 12 10,2  | LEAL
 12 10,1  | LEDA
 12 10,1  | LIDA
 12 10,1  | LILA
 12 11,2  | LIS
 12  9,4  | RIDGE
 12  2,1  | RILL
 11  4,1  | AGED
 11  4,1  | ALGID
 11  4,2  - DIEL
 11  7,7  | ED
 11  7,7  | EL
 11  4,2  - GEE
 11  4,2  - GIE
 11  9,3  | GRID
 11  8,4  | JE
 11  4,2  - LEED
 11  4,2  - LIED
 11  2,8  | QI
 11  3,7  | YOGI
 10 10,4  | AGE
 10  4,1  | AGIEL
 10 10,4  | AIDE
 10 10,4  | AIEL
 10 10,4  | AILE
 10 10,4  | ALD
 10 10,4  | ALII
 10 10,4  | ALL
 10 10,1  | DEIA
 10  5,2  - DELL
 10  9,3  | DREG
 10 10,3  | EADI
 10  9,3  - ED
 10  9,5  - ED
 10 10,1  | EILA
 10  9,3  - EL
 10  9,5  - EL
 10 10,1  | ELIA
 10 10,5  - GI
 10  9,2  | GIRD
 10  9,2  | GIRL
 10  4,5  | GLOP
 10 10,1  | IDEA
 10  4,2  - IDEE
 10  3,3  - IDIC
 10 10,1  | ILEA
 10 10,1  | ILIA
 10  4,6  | LOPED
 10  9,4  | REDIG
 10  2,1  | REG
 10  9,4  | REGD
 10  2,1  | REID
 10  9,4  | RELIG
 10  2,1  | RIDE
 10  2,1  | RIEL
 10  2,1  | RIG
 10  9,4  | RIGEL
 10  9,4  | RIGID
 10  2,1  | RILE
 10  9,4  | RILLE
 10  2,1  | RLD
 10  3,1  | SELLI
 10  3,1  | SIDLE
  9  4,1  | ADIEL
  9  4,1  | AIEL
  9  4,1  | AILED
  9  4,1  | ALLIE
  9  4,2  - DEE
  9  5,2  - DEIL
  9  5,2  - DELE
  9  5,2  - DELI
  9  4,2  - DIE
  9  9,2  | DIRL
  9  9,3  | ERG
  9  9,3  | GRE
  9  4,2  - IDE
  9  7,4  | IDE
  9 11,3  | ISE
  9  4,2  - LEE
  9  4,2  - LIE
  9  9,4  | RILED
  9  9,4  | RILL
  9  3,1  | SLIDE
  9  3,7  | YODE
  8 10,4  | AID
  8 10,4  | AIL
  8  4,1  | AILIE
  8 10,4  | ALE
  8  4,1  | ALLE
  8 10,2  | DEA
  8  5,2  - DEL
  8 10,5  - DI
  8 10,2  | DIA
  8  2,5  - DILLi
  8  4,6  | DOPE
  8  9,3  | DRIE
  8 10,2  | ELA
  8  9,2  | GIRE
  8  5,5  - GiD
  8  3,2  - ID
  8 10,2  | ILA
  8 10,2  | LEA
  8 10,5  - LI
  8  4,6  | LOPE
  8  4,7  | OPED
  8  2,1  | RED
  8  9,4  | REID
  8  2,1  | REL
  8  2,1  | RID
  8  9,4  | RIEL
  8  2,1  | RLE
  8  3,1  | SEG
  8  3,1  | SELD
  8  3,1  | SELL
  8  3,1  | SIG
  8 11,4  | SIGIL
  8  3,1  | SILD
  8  3,1  | SILL
  8  3,7  | YOD
  8  3,7  - YOUSE
  7  4,1  | AIDE
  7  4,1  | AILE
  7  4,1  | ALII
  7  5,2  - DEE
  7  5,2  - DEI
  7  3,5  - DELi
  7  9,2  | DIRE
  7  9,2  | DLR
  7  4,6  | DOP
  7  6,2  - EDGE
  7  9,3  | IRED
  7  9,3  | IRID
  7  9,2  | LIRE
  7  4,6  | LOP
  7  5,5  - LiD
  7  6,6  - NELL
  7  6,6  - NILL
  7  9,4  | RIDE
  7  9,4  | RILE
  7  9,4  | RLD
  7 11,4  | SELLI
  7  3,1  | SIDE
  7  3,1  | SIDI
  7 11,4  | SIDLE
  7  3,1  | SILE
  7  3,1  | SLD
  7  3,1  | SLED
  7  3,1  | SLID
  7 11,4  | SLIDE
  6  4,1  | ALG
  6 10,3  | DAG
  6  5,2  - DE
  6  5,2  - EEL
  6  6,2  - EILD
  6  6,2  - ELLE
  6 11,3  | ES
  6 10,3  | GAD
  6 10,3  | GAL
  6 11,3  | IS
  6 10,3  | LAG
  6  6,6  - NEIL
  6  6,6  - NIDE
  6  6,6  - NIDI
  6  6,6  - NIG
  6  6,6  - NILE
  6  4,7  | OPE
  6  9,4  | REG
  6  2,1  | RIE
  6  9,4  | RIG
  6  9,4  | RLE
  6  3,1  | SED
  6  3,1  | SEID
  6  3,1  | SEL
  6 11,4  | SELD
  6 11,4  | SELL
  6  3,1  | SID
  6  3,1  | SIL
  6 11,4  | SILD
  6 11,4  | SILL
  6 11,4  | SLED
  6 11,4  | SLID
  6  5,1  | TD
  5  4,1  | AGE
  5  4,1  | ALD
  5  4,1  | ALL
  5 10,3  | DAL
  5  9,2  | DER
  5  9,2  | DIR
  5  6,2  - EIDE
  5  6,2  - ELD
  5  6,2  - ELL
  5 10,3  | GAE
  5  7,4  | ID
  5  9,3  | IRE
  5 10,3  | LAD
  5  9,2  | LER
  5  9,2  | LIR
  5  6,6  - NED
  5  6,6  - NID
  5  6,6  - NIL
  5  9,4  | RED
  5  9,4  | REL
  5  9,4  | RID
  5 11,4  | SEG
  5 11,4  | SEID
  5 11,4  | SIDE
  5 11,4  | SIDI
  5 11,4  | SIG
  5 11,4  | SILE
  5 11,4  | SLD
  5  6,5  - iD
  4  4,1  | AG
  4 10,4  | AG
  4  4,1  | AID
  4  4,1  | AIL
  4  4,1  | ALE
  4  5,2  - EE
  4  6,2  - EEL
  4  9,2  | EIR
  4  6,2  - ELI
  4 10,3  | LAI
  4  6,6  - NEI
  4  2,1  | RE
  4  9,4  | RIE
  4 11,4  | SED
  4  3,1  | SEI
  4 11,4  | SEL
  4 11,4  | SID
  4  3,1  | SIE
  4 11,4  | SIL
  4  5,1  | TE
  3  4,1  | AD
  3 10,4  | AD
  3  4,1  | AL
  3 10,4  | AL
  3 10,3  | DA
  3  4,5  - DEi
  3  1,1  | ED
  3  6,2  - ED
  3  1,1  | EL
  3  6,2  - EL
  3  4,5  - ELi
  3  9,3  | ER
  3  5,5  - Gi
  3 10,3  | LA
  3  4,5  - LEi
  3  6,6  - NE
  3  6,6  - NI
  3  9,4  | RE
  3 11,4  | SEI
  3 11,4  | SIE
  2  4,1  | AE
  2 10,4  | AE
  2  4,1  | AI
  2 10,4  | AI
  2  5,5  - Di
  2  1,1  | EE
  2  6,2  - EE
  2  4,5  - IIi
  2  5,5  - Li
  2  3,1  | SI
  2 11,4  | SI
