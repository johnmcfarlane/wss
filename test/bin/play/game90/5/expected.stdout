 43  1,9  - GID
 40  1,9  - LID
 40  2,1  | REGILD
 40  2,1  | RIDGEL
 40  2,1  | RIDGIL
 39 11,1  | GEDS
 39 11,1  | GELS
 39 11,1  | GIDS
 36 11,1  | GIES
 36 10,2  | GLADE
 36  2,1  | RILLED
 33 11,1  | DIGS
 33 11,1  | LEGS
 33  3,7  | YODEL
 33  3,7  | YODLE
 30 11,1  | DELS
 30 11,1  | LIDS
 27 11,1  | DIES
 27 11,1  | LEIS
 27 11,1  | LIES
 24  1,2  - DEIL
 24 11,1  | EGIS
 24 11,1  | ELDS
 24 11,1  | ELLS
 24 10,3  | GALLIED
 24  2,9  - ID
 24 11,1  | ILLS
 23  1,9  - GI
 22 10,3  | GALLED
 22  4,5  - GLiDE
 21  1,1  | EDGE
 21  1,1  | EDILE
 21  1,1  | ELIDE
 21 11,1  | IDES
 21 11,3  | ISLED
 21 11,2  | LISLE
 20  9,3  | GRILLED
 20  1,9  - LI
 20  6,6  - NILLED
 19  3,9  - DE
 18 10,4  | ALGID
 18 10,4  | ALLIED
 18 10,1  | GLIAL
 18 10,1  | LEGAL
 18 10,3  - LI
 18  9,1  | LIER
 18  6,6  - NIELLI
 17  1,2  - LEI
 16 10,4  | AGILE
 16 10,3  | GALL
 16 10,2  | GLAD
 16  9,3  | GRILLE
 16 10,3  | LADLE
 16  2,1  | RIDGE
 16  9,4  | RIDGEL
 16  9,4  | RIDGIL
 16  2,1  | RIGID
 15  1,1  | EIDE
 15  1,1  | ELD
 15  7,7  | ELD
 15  1,1  | ELL
 15  7,7  | ELL
 15  9,3  | GRILL
 15 11,3  | ISLE
 15  8,4  | JELL
 15  8,4  | JILL
 14 10,4  | AGED
 14 10,4  | AILED
 14  6,2  - EDILE
 14 10,2  | EGAD
 14 10,2  | EGAL
 14  6,2  - ELIDE
 14 10,3  | GADI
 14 10,3  | GAED
 14 10,3  | GALE
 14  3,5  - GELiD
 14  9,2  | GIRDLE
 14 10,1  | GLIA
 14 10,1  | IDEAL
 14 10,1  | ILEAL
 14 10,1  | ILIAD
 14 10,1  | ILIAL
 14  8,4  | JIG
 14  9,4  | REGILD
 14  2,1  | RILED
 14  2,1  | RILLE
 14  9,4  | RILLED
 13  1,2  - DE
 13  9,3  | DRILL
 13  4,2  - GIED
 13  9,2  | GIRLIE
 13  9,3  | GRIDE
 13  4,2  - LIEGE
 13  3,1  | SIGIL
 12  4,1  | AGILE
 12 10,3  | DALE
 12 10,2  | DEAL
 12 10,2  | DIAL
 12  9,2  | DIRGE
 12 11,2  | DIS
 12 11,2  | EDS
 12  1,1  | EEL
 12 11,2  | ELS
 12 11,2  | IDS
 12 10,3  | LADE
 12 10,3  | LAID
 12 10,2  | LEAD
 12 10,2  | LEAL
 12 11,2  | LES
 12 11,2  | LIS
 12  9,4  | RIDGE
 12  2,1  | RILL
 11  4,1  | AGED
 11  4,1  | ALGID
 11  4,2  - DIEL
 11  7,7  | ED
 11  7,7  | EL
 11  4,2  - GIE
 11  9,3  | GRID
 11  8,4  | JE
 11  4,2  - LIED
 11  2,8  | QI
 11  3,7  | YOGI
 10 10,4  | AGE
 10 10,4  | AIDE
 10 10,4  | ALL
 10  9,3  | DREG
 10  9,5  - ED
 10  9,3  - EL
 10  9,5  - EL
 10 10,5  - GI
 10  9,2  | GIRD
 10  9,2  | GIRL
 10  4,5  | GLOP
 10 10,1  | IDEA
 10 10,1  | ILEA
 10 10,1  | ILIA
 10  4,6  | LOPED
 10  2,1  | REG
 10  2,1  | RIDE
 10  2,1  | RIEL
 10  2,1  | RIG
 10  9,4  | RIGID
 10  2,1  | RILE
 10  9,4  | RILLE
 10  3,1  | SIDLE
  9  4,1  | AILED
  9  4,2  - DIE
  9  9,2  | DIRL
  9  9,3  | ERG
  9  4,2  - LIE
  9  9,4  | RILED
  9  9,4  | RILL
  9  3,1  | SLIDE
  8 10,4  | AID
  8 10,4  | AIL
  8 10,4  | ALE
  8  4,6  | DOPE
  8  5,5  - GiD
  8  3,2  - ID
  8 10,2  | LEA
  8 10,5  - LI
  8  4,6  | LOPE
  8  4,7  | OPED
  8  2,1  | RED
  8  2,1  | RID
  8  9,4  | RIEL
  8  3,1  | SEG
  8  3,1  | SELL
  8 11,4  | SIGIL
  8  3,1  | SILD
  8  3,1  | SILL
  8  3,7  | YOD
  8  3,7  - YOUSE
  7  4,1  | AIDE
  7  3,5  - DELi
  7  9,2  | DIRE
  7  6,2  - EDGE
  7  9,3  | IRED
  7  9,3  | IRID
  7  9,2  | LIRE
  7  9,2  | LIRI
  7  4,6  | LOP
  7  5,5  - LiD
  7  6,6  - NILL
  7  9,4  | RIDE
  7  9,4  | RILE
  7  3,1  | SIDE
  7 11,4  | SIDLE
  7  3,1  | SLED
  7  3,1  | SLID
  7 11,4  | SLIDE
  6 10,3  | DAG
  6 11,3  | ES
  6 10,3  | GAD
  6 10,3  | GAL
  6 11,3  | IS
  6 10,3  | LAG
  6  6,6  - NEG
  6  6,6  - NIDE
  6  6,6  - NIDI
  6  4,7  | OPE
  6  9,4  | REG
  6  9,4  | RIG
  6  3,1  | SEL
  6 11,4  | SELL
  6 11,4  | SILD
  6 11,4  | SILL
  6 11,4  | SLED
  6 11,4  | SLID
  5  4,1  | AGE
  5  4,1  | ALL
  5 10,3  | DAL
  5  6,2  - EIDE
  5  6,2  - ELD
  5  6,2  - ELL
  5 10,3  | GAE
  5  7,4  | ID
  5  9,3  | IRE
  5 10,3  | LAD
  5  6,6  - NIL
  5  9,4  | RED
  5  9,4  | RID
  5 11,4  | SEG
  5 11,4  | SIDE
  5  6,5  - iD
  4  4,1  | AG
  4 10,4  | AG
  4  4,1  | AID
  4  4,1  | AIL
  4  4,1  | ALE
  4  6,2  - EEL
  4  2,1  | RE
  4  3,1  | SEI
  4 11,4  | SEL
  3  4,1  | AD
  3 10,4  | AD
  3  4,1  | AL
  3 10,4  | AL
  3  1,1  | ED
  3  6,2  - ED
  3  1,1  | EL
  3  6,2  - EL
  3  9,3  | ER
  3  5,5  - Gi
  3 10,3  | LA
  3  4,5  - LEi
  3  6,6  - NE
  3  9,4  | RE
  3 11,4  | SEI
  2  4,1  | AE
  2 10,4  | AE
  2  4,1  | AI
  2 10,4  | AI
  2  5,5  - Li
  2  3,1  | SI
  2 11,4  | SI
